tests/test_models.py

```diff
@@ -1108,6 +1112,82 @@ def test_get_course_and_course_run_no_content_items(self, contains_courses_mock)
         enterprise_customer_catalog = factories.EnterpriseCustomerCatalogFactory()
         assert enterprise_customer_catalog.get_course_and_course_run('fake-course-run-id') == (None, None)
 
+    @mock.patch('enterprise.rules.crum.get_current_request')
+    @mock.patch('enterprise.models.EnterpriseCatalogApiClient')
+    @mock.patch('enterprise.api_client.lms.JwtBuilder', mock.Mock())
+    def test_get_course_and_course_run_no_content_items_waffle_flag(self, api_client_mock, get_current_request_mock):
+        """
+        Verify the `get_course_and_course_run` method returns the same tuple with the enterprise catalog flag active.
+        """
+        with override_flag(USE_ENTERPRISE_CATALOG, active=True):
+            # Set up the API client return values
+            api_client_mock.return_value = mock.MagicMock()
+            api_client_mock.return_value.contains_content_items = mock.MagicMock()
```

Comment:
Nit: I think you can save a few LOC by doing 
```python
@mock.patch('enterprise.models.EnterpriseCatalogApiClient', return_value=mock.MagicMock())
```
and
```python
api_client_mock.return_value.contains_content_items.return_value=False
```

Reply:
Good idea, will do